person.rb

```diff
@@ -0,0 +1,32 @@
+require './nameable'
+class Person < Nameable
+  attr_reader :id, :rentals
```

Comment:
- [ ] The `rentals` variable should have `attr_accessor` instead.

Reply:
Please according to this diagram the `rentals` attr is private (read-only) and should not be writable.

![image](https://user-images.githubusercontent.com/81693905/174063101-155f7474-1f91-4fcd-a899-484a3c0d041b.png)